core/main/src/state/cap/permitted_state.rs

```diff
@@ -150,11 +150,15 @@ impl PermissionHandler {
         app_id: &str,
         allow_cached: bool,
     ) -> RippleResponse {
+        if state.open_rpc_state.is_app_excluded(app_id) {
+            return Ok(());
+        }
+
```

Comment:
I actually recently added this for just `device_fetch_and_store`, but moved it here so it works for both cloud and device perms.